tests/functional/send_sms.js

```diff
@@ -22,7 +22,7 @@
    const SELECTOR_400_HEADER = '#fxa-400-header';
    const SELECTOR_400_ERROR = '.error';
    const SELECTOR_LEARN_MORE = 'a#learn-more';
-   const SELECTOR_LEARN_MORE_HEADER = '#tabzilla';
+   const SELECTOR_LEARN_MORE_HEADER = '#websites-notice';
```

Comment:
The link itself navigates to a `#campaigns` fragment identifier. Would it make more sense to look for that one than this one from the body element?